index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Since this is an internal page, let's not index this page. Add this meta tag somewhere in the head:
```
<meta name="robots" content="noindex">
```